src/main/java/it/gov/pagopa/payment/repository/TransactionInProgressRepositoryExt.java

```diff
@@ -9,4 +9,7 @@ public interface TransactionInProgressRepositoryExt {
   UpdateResult createIfExists(TransactionInProgress trx, String trxCode);
   TransactionInProgress findByTrxCodeThrottled(String trxCode);
   void updateTrxAuthorized(String id, Reward reward, List<String> rejectionReasons);
+  TransactionInProgress findByTrxCode(String trxCode);
```

Comment:
lets rename this and findByTrxCodeThrottled in something that will suggest the filter on chargeDate
```suggestion
  TransactionInProgress findByTrxCodeAndTrxChargeDateNotExpired(String trxCode);
```